cli/azd/pkg/contracts/ux.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package contracts
+
+// EndpointEventData is the schema of the payload for an event
+// of the EndpointEventDataType type.
+type EndpointEventData struct {
+	Endpoint string `json:"endpoint"`
+}
+
+// OperationStartEventData is the schema of the payload for an event
+// of the OperationStartEventDataType type.
+type OperationStartEventData struct {
+	Title string `json:"title"`
+	Note  string `json:"note"`
+}
+
+// WarningEventData is the schema of the payload for an event
+// of the WarningEventDataType type.
+type WarningEventData struct {
+	Description string `json:"description"`
+	HidePrefix  bool   `json:"hidePrefix"`
```

Comment:
I think `HidePrefix` was supposed to be left out based on `warning.go`?

Reply:
Oh - okay.  I wasn't 100% sure what the original intention of the code was (I guess I was in in the mindset of the `json.Marshaller` of "all exported fields are written into the serialized object, unless explicitly elided with something like `json:"-"`), but I understand your interpretation (and it also explains why one of the fields was tagged with the name go was going to use by default and the other was not).

Reply:
Ah, you're correct and it was being surfaced. I'm fine either way. We need to clean up these contracts